packages/flutter/lib/src/widgets/scroll_configuration.dart

```diff
@@ -50,20 +50,24 @@ class ScrollBehavior {
     return null;
   }
 
+  static final ScrollPhysics _bouncingPhysics = const BouncingScrollPhysics().applyTo(const RangeMaintainingScrollPhysics());
+  static final ScrollPhysics _clampingPhysics = const ClampingScrollPhysics().applyTo(const RangeMaintainingScrollPhysics());
```

Comment:
Will this new `ScrollPhysics` provide a way to override platform defaults? I see it's `Bouncing` for iOS and `Clamping` for Android, but we do use `Bouncing` for both platforms. Will we be able to override it? I'm thinking about 1 `overridePhysics` parameter that overrides all platform defaults, or even 1 parameter for each supported platform. Great work!

Reply:
This is just the default, you can set your physics to whatever you want, either globally using a ScrollConfiguration widget, or individually on each Scrollable/ScrollView using the physics argument.

Reply:
What I meant is that, if I override a `ListView`'s `physics` with this one:
```dart
ListView(
  physics: const RangeMaintainingScrollPhysics(),
  ...
),
```
I won't be able to achieve the`BouncingScrollPhysics` effect on Android

Reply:
correct, but instead just use `const BouncingScrollPhysics().applyTo(const RangeMaintainingScrollPhysics());` to compose the two physics models together into one.